src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MessagePackSerializer.cs

```diff
@@ -62,6 +65,8 @@ internal static class MessagePackSerializer
     private const int LIMIT_MAX_FIX_ARRAY_LENGTH = 15;
     private const int STRING_SIZE_LIMIT_CHAR_COUNT = (1 << 14) - 1; // 16 * 1024 - 1 = 16383
 
+    private const int MAX_STACK_ALLOC_SIZE = 256;
```

Comment:
nit: `MAX_STACK_ALLOC_SIZE_IN_BYTES`

Below there is some code computing the size for chars I think the new name might make that more self-explanatory.

Reply:
Done.